docs/manual/webhooks/log-events.md

```diff
@@ -0,0 +1,17 @@
+# Log Events
+
+All Rundeck users have access to the Log Events Webhook plugin. The log events plugin allows users to log incoming webhook events to the log4j logger rundeck.webhooks.log file. Users can post any data they might want to the webhook and it will be logged. Because of this, it can act as a debug endpoint. For example, maybe you don't exactly know what data the source will be sending. Using the log events plugin, you could log it and analyze it before using a different plugin to handle it. 
```

Comment:
Log4j2 or log4j?  not sure if it needs to be distinct in this instance. @sjrd218

Reply:
It would probably be good to change it to log4j2.